modules/atomic-widgets/compatibility.php

```diff
@@ -14,7 +14,9 @@ public function register_hooks() {
 	}
 
 	private function add_atomic_dynamic_tags_settings( $settings ) {
-		$settings['atomicDynamicTags'] = $this->convert_dynamic_tags_to_atomic( $settings['dynamicTags']['tags'] );
+		if ( isset( $settings['dynamicTags']['tags'] ) ) {
```

Comment:
Move to another PR?

(I ❤️ linear b)

Reply:
https://github.com/elementor/elementor/pull/28501